src/Symfony/Component/Worker/Command/WorkerRunCommand.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Symfony\Component\Worker\Command;
+
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Worker\Loop\ConfigurableLoopInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class WorkerRunCommand extends Command
+{
+    private $container;
+    private $processTitlePrefix;
+    private $workers;
+
+    /**
+     * @param ContainerInterface $container          A PSR11 container from which to load workers by names
+     * @param string             $processTitlePrefix
+     * @param string[]           $workers
+     */
+    public function __construct(ContainerInterface $container, $processTitlePrefix, array $workers = array())
+    {
+        parent::__construct();
+
+        $this->container = $container;
+        $this->processTitlePrefix = $processTitlePrefix;
+        $this->workers = $workers;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('worker:run')
+            ->setDescription('Runs a worker')
+            ->setDefinition(array(
+                new InputArgument('worker', InputArgument::REQUIRED, 'The worker'),
+                new InputOption('name', null, InputOption::VALUE_REQUIRED, 'A name, useful for stats/monitoring. Defaults to worker name.'),
+            ))
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $workerName = $input->getArgument('worker');
```

Comment:
Are you sure? If `symfony/symfony` requires `symfony/worker`, `symfony/symfony` should not be installable without filling all `symfony/worker`'s requirements, right?